operations/enterprises.md

```diff
@@ -21,7 +36,17 @@ Returns all company profiles of the enterprise, possible filtered by their ident
 | `ClientToken` | string | required | Token identifying the client application. |
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
-| `Ids` | array of string | optional | If specified, returns only companies with the specified identifiers. |
+| `Ids` | array of string | optional | If specified, filter [Companies](enterprises.md#company) by identifiers. |
```

Comment:
[Companies] is incorrect, this is reference for the entity. Check usage in other description.
[Company](...)s.

Reply:
I told it to wrongly. It doesn't make sense. For company has to be companies. I am sorry. I will fix it directly in the master.